### ISSUE TITLE
Cutmix, Crop, Duplicated 실험 내용

### ISSUE THREAD

**Issue by sangmandu** [sangmandu/2022-DCC#23]
## 공통 인자
{'outdir': 'output', 'datadir': '../data', 'model_name': 'BaseModel', 'use_wandb': True, 'resize': 64, 'batch_size': 512, 'epochs': 25, 'fold': False, 'lr': 0.01, 'optimizer': 'Adam', 'scheduler': 'CyclicLR', 'criterion': 'cross_entropy', 'val_ratio': 0.2, 'test_ratio': 0.0, 'checkpoint': '', 'aug': False, 'sampling': '', 'check_stat': False, 'save_limit': 2, 'seed': 0}

## Illustraton and Real Images with Cutmix
> python classify.py --outdir output --datadir ../data --model_name BaseModel --only_illust False --use_wandb

25503 data has been set
[Train] f1 : 0.64253, best f1 : 0.64253 || acc : 64.25280%, best acc: 64.25280% || loss : 1.2134, best loss: 1.2134 ||
[Valid] f1 : 0.68467, best f1 : 0.68467 || acc : 68.20312%, best acc: 68.20312% || loss : 1.0065, best loss: 1.0065 ||

> python classify.py --outdir output --datadir ../data --model_name BaseModel --only_illust False --use_wandb --cutmix 0.3

25503 data has been set
[Train] f1 : 0.47256, best f1 : 0.49239 || acc : 47.25561%, best acc: 49.23878% || loss : 2.0792, best loss: 1.8814 ||
[Valid] f1 : 0.51433, best f1 : 0.59361 || acc : 51.23047%, best acc: 59.14062% || loss : 1.7911, best loss: 1.4893 ||

> python classify.py --outdir output --datadir ../data --model_name BaseModel --only_illust False --use_wandb --cutmix 0.5

25503 data has been set
[Train] f1 : 0.46184, best f1 : 0.46184 || acc : 46.18389%, best acc: 46.18389% || loss : 2.0719, best loss: 2.0476 ||
[Valid] f1 : 0.52905, best f1 : 0.53452 || acc : 52.69531%, best acc: 53.24219% || loss : 1.6677, best loss: 1.5856 ||

> python classify.py --outdir output --datadir ../data --model_name BaseModel --only_illust False --use_wandb --cutmix 0.8

25503 data has been set
[Train] f1 : 0.37655, best f1 : 0.37715 || acc : 37.65525%, best acc: 37.71534% || loss : 2.2955, best loss: 2.2627 ||
[Valid] f1 : 0.44671, best f1 : 0.45434 || acc : 44.49219%, best acc: 45.25391% || loss : 2.049, best loss: 1.9764 ||

## Cutmix
> python classify.py --outdir output --datadir ../data --model_name BaseModel --save_name only_illust --use_wandb

23578 data has been set
[Train] f1 : 0.67909, best f1 : 0.67909 || acc : 67.90907%, best acc: 67.90907% || loss : 1.1146, best loss: 1.1146 ||
[Valid] f1 : 0.70649, best f1 : 0.7311 || acc : 64.51172%, best acc: 66.97266% || loss : 0.98507, best loss: 0.91784 ||

> python classify.py --outdir output --datadir ../data --model_name BaseModel --save_name only_illust --use_wandb --cutmix 0.3

23578 data has been set
[Train] f1 : 0.51362, best f1 : 0.52311 || acc : 51.36176%, best acc: 52.31120% || loss : 1.8135, best loss: 1.8135 ||
[Valid] f1 : 0.55076, best f1 : 0.60334 || acc : 50.25391%, best acc: 55.29297% || loss : 1.6277, best loss: 1.4036 ||

> python classify.py --outdir output --datadir ../data --model_name BaseModel --save_name only_illust --use_wandb --cutmix 0.5

23578 data has been set
[Train] f1 : 0.45405, best f1 : 0.46533 || acc : 45.40473%, best acc: 46.53320% || loss : 2.0702, best loss: 2.0186 ||
[Valid] f1 : 0.53563, best f1 : 0.57088 || acc : 48.88672%, best acc: 52.26563% || loss : 1.6429, best loss: 1.5663 ||

>python classify.py --outdir output --datadir ../data --model_name BaseModel --save_name only_illust --use_wandb --cutmix 0.8

23578 data has been set
[Train] f1 : 0.4005, best f1 : 0.40424 || acc : 40.04991%, best acc: 40.42426% || loss : 2.1836, best loss: 2.1791 ||
[Valid] f1 : 0.50077, best f1 : 0.50077 || acc : 45.54688%, best acc: 45.54688% || loss : 1.7663, best loss: 1.7663 ||

## Crop and Cutmix
✏️ 23511 of total 23578 images has cropped.

> python classify.py --outdir output --datadir ../data --model_name BaseModel --save_name crop_cutmix --use_crop --use_wandb

23578 data has been set
[Train] f1 : 0.62218, best f1 : 0.62218 || acc : 62.21788%, best acc: 62.21788% || loss : 1.3325, best loss: 1.3325 ||
[Valid] f1 : 0.65709, best f1 : 0.66037 || acc : 60.15625%, best acc: 60.70312% || loss : 1.1753, best loss: 1.1753 ||

> python classify.py --outdir output --datadir ../data --model_name BaseModel --save_name crop_cutmix --use_crop --use_wandb --cutmix 0.3

23578 data has been set
[Train] f1 : 0.49376, best f1 : 0.50575 || acc : 49.37609%, best acc: 50.57509% || loss : 1.8576, best loss: 1.8576 ||
[Valid] f1 : 0.54515, best f1 : 0.60144 || acc : 49.76562%, best acc: 55.17578% || loss : 1.6792, best loss: 1.527 ||

> python classify.py --outdir output --datadir ../data --model_name BaseModel --save_name crop_cutmix --use_crop --use_wandb --cutmix 0.5

23578 data has been set
[Train] f1 : 0.44195, best f1 : 0.4554 || acc : 44.19488%, best acc: 45.54036% || loss : 2.1108, best loss: 2.0265 ||
[Valid] f1 : 0.50692, best f1 : 0.54236 || acc : 46.30859%, best acc: 49.41406% || loss : 1.7583, best loss: 1.7191 ||

> python classify.py --outdir output --datadir ../data --model_name BaseModel --save_name crop_cutmix --use_crop --use_wandb --cutmix 0.8

23578 data has been set
[Train] f1 : 0.38694, best f1 : 0.39366 || acc : 38.69358%, best acc: 39.36632% || loss : 2.1944, best loss: 2.1809 ||
[Valid] f1 : 0.48895, best f1 : 0.48895 || acc : 44.51172%, best acc: 44.51172% || loss : 1.8598, best loss: 1.8598 ||

## Discard duplicated Images
> python classify.py --outdir output --datadir ../data --model_name BaseModel --save_name dup --use_wandb --dup_sim 0.9

9942 data has been set
[Train] f1 : 0.37214, best f1 : 0.37214 || acc : 37.21354%, best acc: 37.21354% || loss : 1.9313, best loss: 1.9272 ||
[Valid] f1 : 0.41275, best f1 : 0.41275 || acc : 40.13672%, best acc: 40.13672% || loss : 1.7872, best loss: 1.7872 ||

> python classify.py --outdir output --datadir ../data --model_name BaseModel --save_name dup --use_wandb --dup_sim 0.85

6502 data has been set
[Train] f1 : 0.32188, best f1 : 0.32188 || acc : 32.18750%, best acc: 32.18750% || loss : 2.1501, best loss: 2.1501 ||
[Valid] f1 : 0.3495, best f1 : 0.36138 || acc : 29.42708%, best acc: 30.33854% || loss : 2.0695, best loss: 2.068 ||

> python classify.py --outdir output --datadir ../data --model_name BaseModel --save_name dup --use_wandb --dup_sim 0.8

6970 data has been set
[Train] f1 : 0.2209, best f1 : 0.22559 || acc : 22.08984%, best acc: 22.55859% || loss : 2.3506, best loss: 2.3506 ||
[Valid] f1 : 0.28876, best f1 : 0.28876 || acc : 26.30208%, best acc: 26.30208% || loss : 2.2332, best loss: 2.2332 ||

> python classify.py --outdir output --datadir ../data --model_name BaseModel --save_name dup --use_wandb --dup_sim 0.75

3508 data has been set
[Train] f1 : 0.13281, best f1 : 0.18086 || acc : 13.28125%, best acc: 18.08594% || loss : 2.7282, best loss: 2.6465 ||
[Valid] f1 : 0.17227, best f1 : 0.23938 || acc : 10.93750%, best acc: 15.33203% || loss : 2.6462, best loss: 2.5643 ||

> python classify.py --outdir output --datadir ../data --model_name BaseModel --save_name dup --use_wandb --dup_sim 0.7

3721 data has been set
[Train] f1 : 0.1625, best f1 : 0.17227 || acc : 16.25000%, best acc: 17.22656% || loss : 2.5068, best loss: 2.4827 ||
[Valid] f1 : 0.1663, best f1 : 0.17567 || acc : 11.71875%, best acc: 12.30469% || loss : 2.4103, best loss: 2.3918 ||
